src/backend/commands/copyfrom.c

```diff
@@ -565,6 +565,19 @@ CopyMultiInsertBufferCleanup(CopyMultiInsertInfo *miinfo,
 	for (i = 0; i < MAX_BUFFERED_TUPLES && buffer->slots[i] != NULL; i++)
 		ExecDropSingleTupleTableSlot(buffer->slots[i]);
 
+	if (AMHandlerIsAO(buffer->resultRelInfo->ri_RelationDesc->rd_amhandler))
```

Comment:
Is it possible PAX and storage am still fail?

Reply:
Will have a test later

Reply:
@jiaqizho has a test, confirm pax  has this issue too.

![img_v3_02di_1ee93136-f9da-4daf-bab0-5b2427e9518g](https://github.com/user-attachments/assets/efa74c65-7f8d-4adf-94e8-740fee14b0cd)

Reply:
Fixed.